### PR TITLE
AS-541: upgrade to sbt 1.4.4 [risk: low]

### PR DESCRIPTION
**Pull request by davidangb**
Upgrades the main orchestration build to use sbt latest version (1.4.4).

We should also upgrade the automation project to use sbt latest and Scala 2.12-latest; I'd like to do that as a separate PR to keep things clean.

See explanatory comments inline to the code. Anything not explained is purely for syntax compatibility.

--------

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [x] I've followed [the instructions](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [x] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [x] I've updated the [FISMA documentation](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,7 @@ config/
 
 # sbt specific
 .sbtopts
+.bsp/
```
Comment (davidangb) [inline .gitignore:8]: sbt 1.4.x now supports BSP: https://contributors.scala-lang.org/t/build-server-protocol-in-sbt/4234/27 so we need to ignore this generated directory.

File: project/plugins.sbt [hunk 2]
```diff
@@ -1,9 +1,5 @@
...
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.15.0")
 
-addSbtPlugin("io.spray" % "sbt-revolver" % "0.7.2")
+addSbtPlugin("io.spray" % "sbt-revolver" % "0.9.1")
 
-addSbtPlugin("com.github.sbt" % "sbt-jacoco" % "3.2.0")
-
-addSbtPlugin("com.github.tkawachi" % "sbt-repeat" % "0.1.0")
-
-addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.9.2")
```
Comment (davidangb) [inline project/plugins.sbt:9]: `sbt-repeat` and `sbt-dependency-graph` were only here to assist with occasional debugging/developer goodness tasks. Let's not include them by default; individual devs can modify plugins as needed during debugging sessions.

The `sbt-jacoco` is not strictly necessary as part of this sbt upgrade but felt worth it.

File: project/plugins.sbt [hunk 3]
```diff
@@ -1,9 +1,5 @@
...
\ No newline at end of file
+addSbtPlugin("com.github.sbt" % "sbt-jacoco" % "3.3.0")
```
Comment (davidangb) [inline project/plugins.sbt:5]: * `sbt-assembly` and `sbt-revolver` upgrades are necessary for sbt 1.4 compatibility.
* `sbt-repeat` and `sbt-dependency-graph` were only here to assist with occasional debugging/developer goodness tasks. Let's not include them by default; individual devs can modify plugins as needed during debugging sessions.
* `sbt-jacoco`: this upgrade is not strictly necessary as part of this sbt upgrade but felt worth it.

**Comment by davidangb** [comment]
unrelated fiab-start failure:
```
10:20:41 Pulling bond-app (quay.io/databiosphere/bond:develop)...
10:20:43 Get https://quay.io/v2/databiosphere/bond/manifests/develop: received unexpected HTTP status: 502 Bad Gateway
10:20:43 Build step 'Execute shell' marked build as failure
```

so, jenkins retest.

**Review by MatthewBemis — APPROVED** [review]
:shipit:

**Pull request merged**